docs/posts/wow-analysis/index.qmd

```diff
@@ -0,0 +1,174 @@
+---
+title: "Analsis of World of Warcraft Data"
+author: "Tyler White"
+error: false
+date: "2024-02-29"
+categories:
+  - blog
+  - data engineering
+  - duckdb
+---
+## Introduction
+This dataset has been around for a while, but it’s newer to me. I grew up playing video games, and I always had a great time jumping into Azeroth and whatever world was introduced in later expansions with my friends.
```

Comment:
Can you wrap lines? We've got a docs style guide here: https://ibis-project.org/contribute/03_style#documentation-blog-and-other-prose-style